Cargo.toml

```diff
@@ -1,20 +1,21 @@
 [package]
 name = "kuchiki"
-version = "0.7.3"
+version = "0.8.0"
```

Comment:
Does this PR include breaking changes? If I remember correctly all dependencies updated here are private.

Reply:
I believe there are breakingchanges to src/select.rs#207 related to changing version of CSS selectors

Before we had `local_name(&self) -> &LocalName`, now there is a `has_local_name(&self, &LocalName) -> bool`

Reply:
But that’s an impl of a trait from a private dependency. It’s not exposed to users (or at least it’s not intended to).

Reply:
https://github.com/kuchiki-rs/kuchiki/blob/62110940e56b9ef462e294087ae07c057c45382f/src/tree.rs#L54

NodeData is exposed through the public API, and QualName comes from the public html5ever API.

Reply:
Ah, good point. Looks like I either remembered wrong about html5ever being a private dependency, or made that mistake from the beginning.

Reply:
@SimonSapin should a new issue for this be raised (having html5ever as a public dependency)?